must-gather/node-gather/clusterrole.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: perf-node-gather-pods-reader
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get", "watch", "list"]
```

Comment:
list and get - fine. But do we need to watch pods?

Reply:
Not really but I though it wouldnot hurt being just a read command.
will change it

Reply:
it's not a deal breaker, but as general approach let's always try to minimize the permissions we grant